Resources/Locale/en-US/_NF/m_emp/m_emp.ftl

```diff
@@ -17,3 +18,13 @@ m_emp-system-generator-examined-cooling-down = Cooling down.
 m_emp-system-generator-examined-recharging = Recharging. Ready in: {$timeLeft} seconds.
 m_emp-system-generator-delay-upgrade = Cooling down / Recharging speed
 
+# M_EMP Console
+m_emp-console-menu-title = M_EMP
+m_emp-menu-note0 = Send a request to discharge.
+m_emp-menu-note1 = ATTENTION: Firing this weapon will produce
+m_emp-menu-note2 = an electro-magnetic pulse capable of disabling
+m_emp-menu-note3 = all ships within a large radius.
+m_emp-menu-note4 = By discharging this weapon,
+m_emp-menu-note5 = you agree to be held accountable.
```

Comment:
hm instead of splitting it all up in a bunch of lines, it might be better formatted into one text block and wordwrapped by your UI container instead.  That way, translation texts will also look the same without breaking the UI.